### ISSUE TITLE
Comparing string content, more than just string length?

### ISSUE THREAD

**Issue by bnb012008** [GoogleCloudPlatform/professional-services-data-validator#486]
Is there any way to do this in the existing code?  Has anyone thought about an addition to the existing code?  Please contact me.

**Comment by renzokuken** [comment]
@bnb012008 could you elaborate more on what you are trying to achieve? Row level validation lets you compare the raw contents of columns regardless of their datatype. String length is calculated by default in order to be passed to an aggregate function as a "summary" of validating string quality e.g. a SUM(len(string)) can quickly identify truncation or similar.

https://github.com/GoogleCloudPlatform/professional-services-data-validator#row-validations

Please let me know if you have any questions.

**Comment by bnb012008** [comment]
Is it okay to prepare and share a 'view only' google doc link?

**Comment by nehanene15** [comment]
It would be better to post your question or comments in this issue itself. You can run row level validation on string fields which will check for an exact match with comparison fields or a checksum.

**Comment by bnb012008** [comment]
Source system DB2, column type varchar, target system BQ, created via an AVRO file in a GS bucket, target column string.  Number and string length compares generally work.  String/varchar content does not compare using the process described.  I produced an initial yaml file and then edited it to correct it mainly case corrections on column names.  I used: data-validation validate row -sc db2edt1d -tc bqv -tbls EDTDM.DIMENSION_DATE=bnb01_test.DIMENSION_DATE -pk 'dt_sk' -rr -rbs '10' -comp-fields 'row_src_id' -c yamlrc01_RS.yaml.  I can provide the -v output that was created with the corrected yaml file if needed.
